android-lib/src/main/java/com/kinvey/android/KinveyHandlerThread.java

```diff
@@ -38,4 +39,16 @@ protected synchronized void onLooperPrepared() {
         }
     }
 
+    public synchronized void stopHandlerThread() {
```

Comment:
@yuliya-guseva Please see if we can reduce the access level of this method to package-level.